tests/remote/integration_google_cloud/outputs.tf

```diff
@@ -0,0 +1,4 @@
+output "test" {
+  value     = opslevel_integration_google_cloud.test
+  sensitive = true
```

Comment:
```suggestion
```
While this technically works it's not clear whether the entirety of this opslevel_integration_google_cloud is sensitive or parts of it. Marking the `private_key` sensitive in the input variables ensures that it's treated as sensitive on the way out of the module

Reply:
@davidbloss with your other suggestion `private_key` has been marked as sensitive, I then used the `nonsensitive()` function to wrap the value as described here - https://support.hashicorp.com/hc/en-us/articles/5175257151891-How-to-output-sensitive-data-with-Terraform